src/components/MoreInformation/MoreInformation.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { MutableRefObject, PropsWithChildren, useEffect, useRef, useState } from 'react'
+
+import styles from './MoreInformation.module.scss'
+
+type Props = {
+  minHeight: number,
+  onToggle: () => void
+}
+
+export const MoreInformation = ({ minHeight, onToggle, children }: PropsWithChildren<Props>) => {
+  const [height, setHeight] = useState(0)
+
+  const ref = useRef<HTMLDivElement | null>(null) as MutableRefObject<HTMLDivElement>
+
+  useEffect(() => {
+    setHeight(ref.current.clientHeight)
```

Comment:
do we really need set new value for state, every time when we open/close ?

Reply:
fixed